### PR TITLE
[refactor] Changes required for web-platform 0.7.0

### PR DESCRIPTION
**Pull request by FallenChromium**
This system doesn't compile with the latest version of `sc-machine` due to the breaking changes introduces in the version 0.7.0. That's a small PR aiming to fix it.

### CODE REVIEW

**Review by MksmOrlov — APPROVED** [review]

File: problem-solver/cxx/inferenceModule/logic/LogicExpression.cpp [hunk 1]
```diff
@@ -158,7 +158,7 @@ std::unique_ptr<LogicExpressionNode> LogicExpression::build(ScAddr const & node)
 
   if (atomicFormulaIter3->Next())
   {
-    SC_LOG_DEBUG(context->HelperGetSystemIdtf(node) + " is a template")
+    SC_LOG_DEBUG(context->HelperGetSystemIdtf(node); + " is a template");
```
Comment (MksmOrlov) [inline problem-solver/cxx/inferenceModule/logic/LogicExpression.cpp:161]:
```suggestion
    SC_LOG_DEBUG(context->HelperGetSystemIdtf(node) + " is a template");
```

Comment (MksmOrlov) [reply problem-solver/cxx/inferenceModule/logic/LogicExpression.cpp:161]: And so on

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

**Comment by MksmOrlov** [comment]
Fixed in https://github.com/ostis-apps/ostis-inference/pull/28

**Comment by FallenChromium** [comment]
Closed in favor of #28

**Pull request closed without merging**